### PR TITLE
doxygen: Depends on flex and bison

### PR DESCRIPTION
**Pull request by saketkc**

### CODE REVIEW

File: Library/Formula/doxygen.rb [hunk 1]
```diff
@@ -21,6 +21,8 @@ class Doxygen < Formula
   depends_on "graphviz" => :optional
   depends_on "qt" if build.with? "doxywizard"
   depends_on "llvm" => "with-clang" if build.with? "libclang"
+  depends_on "flex" if OS.linux?
+  depends_on "bison" if OS.linux?
```
Comment (sjackman) [inline Library/Formula/doxygen.rb]: Please change these both from `if OS.linux?` to `unless OS.mac?`

**Comment by sjackman** [comment]
Hi, Saket. Would you like to fix up this PR?

**Comment by saketkc** [comment]
Fixed

**Comment by sjackman** [comment]
Thanks, Saket.

**Pull request closed without merging**